Assignment/solution_student/index.html

```diff
@@ -4,30 +4,95 @@
     <meta charset="UTF-8">
     <title>Relaxr Blog</title>
     <meta name="description" content="An HTML Starter Template">
-    
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,300,300italic,600,600italic,400italic,800,800italic' rel='stylesheet' type='text/css'>
     <link rel="stylesheet" href="css/normalize.css"><!-- CSS reset: http://necolas.github.io/normalize.css/ -->
     <link rel="stylesheet" href="css/style.css"><!-- Our site's stylesheet -->
 
     <link rel="shortcut icon" type="image/ico" href="images/favicon.png"><!-- the favicon -->
   </head>
   <body>
+    
+    <header>
 
-    <!-- YOUR MARKUP GOES HERE! -->
-    <div class="container">
+      <h1>Relaxr</h1>
+<!--  -->
+        <nav>
+          <ul>
+            <li><a href="#" class="nav-links">About</a></li>
+            <li><a href="#" class="nav-links">FAQ</a></li>
+            <li><a href="#" class="nav-links">Team</a></li>
+            <li><a href="#" class="nav-links">Contact Us</a></li>
+            <li><a href="#" class="nav-links">Blog</a></li>
+          </ul>
+        </nav>
     
-      <header>
-        <h1>FEWD HTML Template</h1>
-      </header>
+    </header>
+
+    <main>
+
+    <div class="main clearfix">
```

Comment:
I think you might have added this and then realized you didn't need it, because it has no closing tag, and it's not used in your CSS, so it'd be best to remove it :)